changes/ocd-3609.sql

```diff
@@ -0,0 +1,138 @@
+DROP FUNCTION IF EXISTS openchpl.add_missing_measure_criteria_mapping(bigint,bigint,text,text,text,boolean) ;
+CREATE FUNCTION openchpl.add_missing_measure_criteria_mapping(origCriterionId bigint, curesCriterionId bigint, uploadValue text, name text, description text, removed boolean)
+  RETURNS void AS $$
+	DECLARE
+	  measureId bigint;
+	BEGIN
+		SELECT DISTINCT measure_id 
+		INTO measureId		
+		FROM openchpl.allowed_measure_criteria
+		WHERE id =
+			(SELECT allowed_criteria_measure_id FROM openchpl.allowed_measure_criteria_legacy_map
+			WHERE macra_criteria_map_id = 
+				(SELECT id FROM openchpl.macra_criteria_map
+				WHERE criteria_id = origCriterionId
+				AND value = uploadValue)
+			);
+		
+		IF measureId IS NULL THEN
+			RAISE NOTICE 'No measure id found for criterion id % and upload value %', origCriterionId, uploadValue;
+		ELSE 
+			-- insert an entry for the cures criterion into the "deprecated" macra measure table
+			INSERT INTO openchpl.macra_criteria_map
+				(criteria_id, value, name, description, removed, last_modified_user)
```

Comment:
I don't see any of the calls to this function passing "false" in for this value. Are _all_ of these measures/mappings/whatever actually removed?

Reply:
Oh, yes they are all removed. The problem was with removed macra measures. I could have not had that parameter but put it in for whatever reason... I don't think it's hurting anything for this temporary function.

Reply:
Works for me. Thanks!